draft-ietf-httpbis-digest-headers.md

```diff
@@ -270,9 +270,14 @@ The relationship between `Content-Location` (see Section 7.8 of
 representation metadata, payload transformations and HTTP methods on Digest is
 provided in {{examples-unsolicited}} and {{examples-solicited}}.
 
-A `Digest` field MAY contain multiple representation-data-digest values. This
-could be useful for responses expected to reside in caches shared by users with
-different browsers, for example.
+A `Digest` field MAY contain multiple representation-data-digest values.
+For example, a server may provide representation-data-digest values using different algorithms,
+allowing it to support a population of clients with different evolving capabilities;
+this is particularly useful in support of transitioning away
+from weaker algorithms should the need arise (see {{algorithm-agility}}).
+Another use case for sending multiple checksums is
+for responses expected to reside in caches shared by users with
+different browsers.
```

Comment:
This sentence seems to me to just repeat the notion of evolving capability already introduced in the paragraph.

I suspect this text, originally in RFC 3230, was attempting to address transparent caching proxies. I'm not convinced we need to keep it in that form given how many HTTP deployments work. Maybe @mnot has an opinion.

Reply:
@LPardue Are you proposing this?

```suggestion
```

Reply:
Yes.